src/AltinnCore/Common/Services/Implementation/InstanceStudioSI.cs

```diff
@@ -63,26 +63,30 @@ public class InstanceStudioSI : IInstance
         public async Task<Instance> InstantiateInstance(StartServiceModel startServiceModel, object serviceModel, IServiceImplementation serviceImplementation)
         {
             Guid instanceId = Guid.NewGuid();
-            string applicationId = startServiceModel.Service;
-            string applicationOwnerId = startServiceModel.Org;
+            string appName = startServiceModel.Service;
+            string org = startServiceModel.Org;
             int instanceOwnerId = startServiceModel.ReporteeID;
 
-            ServiceState currentState = _workflow.GetInitialServiceState(applicationOwnerId, applicationId);
+            ServiceState currentState = _workflow.GetInitialServiceState(org, appName);
 
             Instance instance = new Instance
             {
                 Id = instanceId.ToString(),
                 InstanceOwnerId = instanceOwnerId.ToString(),
-                ApplicationId = applicationId,
+                AppId = appName,
```

Comment:
Is there a reason AppId is set to AppName here? Are we after org/appname or just appname?

Reply:
Well spotted. Have changed to appId = {org}/{appName}.